### PR TITLE
Feature branch

### PR DESCRIPTION
**Pull request by sulaimonaa**
- Create a new repo for this project.
- Create a file named schema_based_on_diagram.sql where you implement the database from the diagram.
- Join tables from many-to-many relationships might not appear in the diagram, but you still need them.
- Added Foreign Key indexes.

### CODE REVIEW

File: schema_based_on_diagram.sql [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+);
+ALTER TABLE medical_histories
+ADD COLUMN treatment_id INT REFERENCES treatments(id);
+-- add indexes
+CREATE INDEX idx_patients ON medical_histories(patient_id ASC);
+CREATE INDEX idx_medical_history_id ON invoices(medical_history_id ASC);
+CREATE INDEX idx_invoices ON invoices(id ASC);
+CREATE INDEX idx_treatment_medical_history ON treatement_medical_history(medical_history_id ASC);
+CREATE INDEX idx_treatment_id ON invoice_items(treatment_id ASC);
+
```
Comment (MahmoudBakr23) [inline schema_based_on_diagram.sql]: - [ ] Great job so far adding the correct INDEXES. However, you might have missed one of them, please create index on  `medical_histories_has_treatments (treatment_id`

File: README.md [hunk 2]
```diff
@@ -1 +1,161 @@
...
+
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
+
+I Would like to Thanks Microverse for providing the Project.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 📝 License <a name="license"></a>
+
+This project is [MIT](MIT.md) licensed.
```
Comment (MahmoudBakr23) [inline README.md:161]: - [x] Please fix the MIT link for a professional README and repo. 🙏

**Review by MahmoudBakr23 — CHANGES_REQUESTED** [review]
Hello TEAM😃, 

Great work and effort you put into this project so far.

## Highlights 🚀 

There are some excellent points that you have made perfectly. Keep up! 💪
- Well done on creating the TABLES of the database according to the diagram
- Descriptive PR

However, there is always room for improvement.

> I'd like to remind you that your long hours of work are appreciated. However, the code review process is for improving your code, not criticizing it. 😄

 ## Required Changes ♻️
_Please check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

N/A

Cheers, and Happy coding!👏👨‍💻💻

Feel free to mention @MahmoudBakr23 to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by mikethreels — APPROVED** [review]
Hi Team, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
- Good job implementing the requested changes.
## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**